honeybee_radiance_folder/folder.py

```diff
@@ -0,0 +1,584 @@
+                raise ValueError('{} already exist.'.format(directory))
+            os.makedirs(directory)
+
+
+class AssetFolder(_FolderCollection):
+    """Input asset folder.
+    
+    This folder includes input assets for daylight model. They can be divided into
+    senders and receivers.
+
+    ├───asset                     [required]
+        ├───grid
+        ├───ies
+        ├───sky
+        ├───sun
+        └───view
+    """
+    ASSET_ROOT = 'asset'
+    GRID = 'asset/grid'
+    IES = 'asset/ies'
+    SKY = 'asset/sky'
+    SUN = 'asset/sun'
+    VIEW = 'asset/view'
+
+    ASSET = ASSET_ROOT, GRID, IES , SKY, SUN, VIEW
+    REQUIRED = (ASSET_ROOT,)
+
+    @property
+    def root(self):
+        """Asset root folder."""
+        return self.ASSET_ROOT
+
+    @property
+    def ies_folder(self):
+        """Electric light ies files folder."""
+        return self.IES
+    
+    def ies_files(self, rel_path=True):
+        """List of ies files."""
+        return self._find_files(self.IES, '.ies', rel_path)
+
+    @property
+    def sky_folder(self):
+        """Sky files folder."""
+        return self.SKY
+
+    def sky_files(self, rel_path=True):
+        """List of sky files."""
+        return self._find_files(self.IES, '.sky', rel_path)
+
+    @property
+    def sun_folder(self):
+        """Sunpath and sun source folder."""
+        return self.SUN
+
+    def sun_files(self, rel_path=True):
+        """List of ies files."""
+        return self._find_files(self.SUN, '.sun', rel_path)
+
+    @property
+    def grid_folder(self):
+        """Sensor grids folder."""
+        return self.GRID
+
+    def grid_files(self, rel_path=True):
+        """List of sensor grid files."""
+        return self._find_files(self.GRID, '.pts', rel_path)
+
+    @property
+    def view_folder(self):
+        """View files folder."""
+        return self.VIEW
+
+    def view_files(self, rel_path=True):
+        """List of view files."""
+        return self._find_files(self.VIEW, '.vf', rel_path)
+
+    def write(self, overwrite=True):
+        """Write model folder."""
+        for subfolder in self.ASSET:
+            directory = os.path.join(self.folder, subfolder)
+            if os.path.exists(directory) and not overwrite:
+                raise ValueError('{} already exist.'.format(directory))
+            os.makedirs(directory)
+
+
+class OutputFolder(_FolderCollection):
+    """Radiance output folder.
+
+    ├───output                    [required]
+        ├───matrix
+        ├───octree
+        ├───postprocess
+        └───temp
+    """
+    OUTPUT_ROOT = 'output'
+    OCTREE = 'output/octree'
+    MATRIX = 'output/matrix'
+    TEMP = 'output/temp'
+    POSTPROCESS = 'output/postprocess'
+    OUTPUT = OUTPUT_ROOT, OCTREE, MATRIX, TEMP, POSTPROCESS
+    REQUIRED = (OUTPUT_ROOT,)
+
+    @property
+    def root(self):
+        """Output root folder."""
+        return self.OUTPUT_ROOT
+
+    @property
+    def octree(self):
```

Comment:
I am guessing that you plan to implement a means of getting the octree files like what you have done with the assets folder?  I'm just making sure that this wasn't something that you forgot to do since I didn't see a TODO about it anywhere in the code.

Reply:
Probably. Added a TODO to class itself. The question is what does folder look up mean for output folders. For input folders it is more restrict but here we should probably support multiple file extensions. I'm not sure yet.